src/Phase.jl

```diff
@@ -72,13 +72,32 @@ function IdealDiluteSolution(species,reactions,solvent; name="",diffusionlimited
 end
 export IdealDiluteSolution
 
-@with_kw struct HomogeneousCatalyst{Q<:AbstractReaction} <: AbstractPhase
+@with_kw struct IdealSurface{W<:Tuple,W2n} <: IdealPhase
```

Comment:
Thanks for the PR! I think `W2n` here should be `W2`?

Reply:
Thanks! Should be fixed now!